scripts/tasks/task-sysex-logging.py

```diff
@@ -0,0 +1,104 @@
+#! /usr/bin/env python3
+import sys
+import time
+import argparse
+import rtmidi
+import sys
+
+
+def argparser():
+    midiout = rtmidi.MidiOut()
+    available_ports = midiout.get_ports()
+    s = "\nusage example: dbt sysex-logging 123"
+    s += "\n\nstart listening to console logs on MIDI port # 123"
+    s += "\n\nAvailable MIDI ports:\n\n"
+    for i, p in enumerate(available_ports):
+        s += "Port # " + str(i) + " : " + str(p) + "\n"
+
+    parser = argparse.ArgumentParser(
+        prog="sysex-logging",
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        description="Listen to console logs via MIDI",
+        epilog=s,
+        exit_on_error=True,
+    )
+    parser.group = "Development"
+    parser.add_argument(
+        "port_number",
+        help="MIDI port number (example: 123) use 'dbt sysex-logging -h' to list available ports",
+        type=int,
+    )
+    return parser
+
+
+def unpack_7bit_to_8bit(bytes):
+    output = bytearray()
+
+    for b in bytes:
+        # Extract 7-bit value
+        value = b & 0x7F
+
+        # Extend to 8-bit
+        if value & 0x40:
+            value |= 0x80
+
+        output.append(value)
+
+        result = []
+        for byte in bytes:
+            result.append(byte & 0x7F)
+            result.append(byte >> 7)
+        return bytearray(result)
+
+
+def sysex_console(port):
+    midiout = rtmidi.MidiOut()
+    midiout.open_port(port)
+
+    with midiout:
+        data = bytearray(6)
+        data[0] = 0xF0  # sysex message
+        data[1] = 0x7D  # deluge (midi_engine.cpp midiSysexReceived)
+        data[2] = 0x03  # debug namespace
+        data[3] = 0x00  # sysex.cpp, sysexReceived
```

Comment:
0x00 0x01 is the command to enable sysex logging - please clarify that in the comment